src/Components/Avatar/utils.js

```diff
@@ -0,0 +1,22 @@
+import { message } from 'antd';
+
+function getBase64(img, callback) {
+    const reader = new FileReader();
+    reader.addEventListener('load', () => callback(reader.result));
+    reader.readAsDataURL(img);
+}
+
+function beforeUpload(file) {
+    const { type, size } = file;
+    const isJpgOrPng = type === 'image/jpeg' || type === 'image/png';
+    if (!isJpgOrPng) {
+        message.error('You can only upload JPG/PNG file!');
+    }
+    const isLt2M = size / 1024 / 1024 < 2;
```

Comment:
Shouldn't this isLt2M constant be a bool function?

Reply:
Could be, I'll change it.